rocketmq-flink/src/main/java/org/apache/rocketmq/flink/RocketMQSink.java

```diff
@@ -62,10 +62,22 @@
     private int batchSize = 1000;
     private List<Message> batchList;
 
+    private int msgDelayLevel = RocketMQConfig.MSG_DELAY_LEVEL00;
```

Comment:
messageDeliveryDelayLevel could be better naming

Reply:
> messageDeliveryDelayLevel could be better naming
sorry for reply late. this is my first pull request, i do not know the modify procedure; 
should i modify and start an pull request?

Reply:
now, you just need to modify it in this branch. more details you could look through google how could pr in github :-)